runner/src/server/plugins/engine/views/partials/form.html

```diff
@@ -43,7 +43,7 @@
 
   {% if not (components[0].type == "FlashCard") %}
     {% if isStartPage %}
-      {{ govukButton({ attributes: { id: "submit" }, text: "Start now", isStartButton: true }) }}
+      {{ govukButton({ attributes: { id: "submit" }, text: "Continue", isStartButton: true }) }}
```

Comment:
this hasnt changed the behaviour in my runner, it still says save and continue on the start page :
<img width="1055" alt="image" src="https://user-images.githubusercontent.com/95699325/184385107-ba9d72ad-60c9-44ff-bb48-5e7ca3f5b0e0.png">

Reply:
page may not be set to a start page maybe?

Reply:
Change the page to Start page from question page

Reply:
I haven't updated the forms json yet so it won't work but you could try creating a new form to test it out?
I had a quick chat with Emma about the continue button layout and she said she is going to get back to us on Monday hence I haven't changed the json yet 🙂